File: Odoo/addons/hackpdm/models/hp_property.py
from odoo import models, fields, api

class hp_property(models.Model):
    _name = 'hp.property'
    _description = 'hp property'
    _inherit = 'hp.common.model'

    name = fields.Char(
        required=True,
        string='property name',
    )
    prop_type = fields.Char(
        string='property type',
    )

    active = fields.Boolean(
        string='active',
    )

File: Src/HackPDM.Infrastructure/Odoo/Models/HpProperty.cs
using HackPDM.Domain.OdooModels;
using HackPDM.Domain.OdooModels.Models;
using HackPDM.Shared.GlobalData;
using HackPDM.Shared.OdooAttributes;

namespace HackPDM.Infrastructure.Odoo.Models;

[OdooModel(OdooDefaultsConstants.HP_PROPERTY_NAME, OdooDefaultsConstants.HP_PROPERTY)]
public partial class HpProperty : HpBaseModelTransport<HpProperty>, IHpPropertyModel
{
	[OdooProp(OdooFieldType.Char, "name")] public string? name { get; set; }
	[OdooProp(OdooFieldType.Char, "prop_type")] public string? prop_type { get; set; }
	[OdooProp(OdooFieldType.Boolean, "active")] public bool? active { get; set; }
}
